WalletWasabi.Fluent/ViewModels/ViewModelBase.cs

```diff
@@ -1,11 +1,56 @@
 using ReactiveUI;
+using System;
+using System.Collections;
+using System.ComponentModel;
+using WalletWasabi.Gui.Validation;
 
 namespace WalletWasabi.Fluent.ViewModels
 {
 	/// <summary>
 	/// Base class for all ViewModels.
 	/// </summary>
-	public class ViewModelBase : ReactiveObject
+	public class ViewModelBase : ReactiveObject, INotifyDataErrorInfo, IRegisterValidationMethod
```

Comment:
Wondered if it is possible to share the existing implementation?

i.e. delete our ViewModelBase... or make our inherit WalletWasabi.Gui.ViewModelBase?

to prevent duplicating code?

Reply:
Done.